tests/cases/fourslash/findAllRefsThisKeywordMultipleFiles.ts

```diff
@@ -12,4 +12,4 @@
 //// // different 'this'
 //// function f(this) { return this; }
 
-verify.singleReferenceGroup("this");
+verify.singleReferenceGroup("module globalThis\nthis: typeof globalThis");

```

Comment:
Hm. We might want a custom way to display the symbol associated with the global scope. Like just `(global) globalThis`?

Reply:
I don't like the current state, but isn't there some value to printing `typeof globalThis` ? It teaches people what type they have to write to refer to the type of globalThis.

Reply:
eh? I don't really feel too strongly about it. It just felt like for something as core as a refernce to the global scope, something a little more clear then `module globalThis; this: typeof globalThis` may be warranted. It's a unique object and so maybe deserves unique output.